_includes/header.html

```diff
@@ -88,19 +88,18 @@
             {% endfor %}
           </ul>
         </li>
-
       </ul>
     </div>
     <div class="usa-nav__secondary">
       <div class="usa-nav__secondary-container">
         <ul class="usa-nav__secondary-links">
-          <li class="usa-nav__secondary-item bold">
+          <li id="sign-in" class="usa-nav__secondary-item bold">
             <span class="font-sans-md">{% t nav.sign_in %}</span>
-            <button tabindex="-1" class="usa-button--small sign-in-logo sign-in-logo-padding">
-              <a href="{% if site.lang != 'en' %}https://secure.login.gov/{{site.lang}}/{% else %}https://secure.login.gov/{% endif %}" title="Sign In" aria-label="Sign in">
-                <img src="{{ site.baseurl }}/assets/img/logo_sign_in.svg" alt="login.gov">
-              </a>
-            </button>
+            <span class="usa-button--small sign-in-logo sign-in-logo-padding">
+              <a href="https://secure.login.gov/{% if site.lang != 'en' %}{{site.lang}}{% endif %}"
+                style="background: url({{ site.baseurl }}/assets/img/logo_sign_in.svg) center/contain no-repeat"
```

Comment:
I feel like it's generally an anti-pattern to do direct `style` attributes in HTML for things we can manage in the stylesheets.

Was the reason for doing this to get the asset URL correct? If so, it looks like other background URLs in our stylesheets seem to do relative URLs, ex `url(../img/alerts/warning.svg)`, have we given that a shot?

Reply:
Thanks for pointing this out - I originally had this rule in a stylesheet but the url path was incorrect.  I should have looked through the other files for a correct example.  I will revert/correct this change.  I agree with you that inline styles are best avoided when possible.